long-lived-testnets/v2-052-1/README.md

```diff
@@ -0,0 +1,25 @@
+
+# `v2-052-1` Chain Details
+
+* **Chain-ID**: `v2-052-1`
+* **denom**: `stake`
+* **Current Cosmos SDK upstream**: [`v0.52.x`](https://github.com/cosmos/cosmos-sdk/tree/release/v0.52.x)
```

Comment:
Can we directly link the simapp of the branch in those docs? it is easy to see what is being built directly.